code/modules/tgui_input/modal/speech.dm

```diff
@@ -0,0 +1,105 @@
+/**
+ * Alters text when players are injured.
+ * Adds text, trims left and right side
+ *
+ * Arguments:
+ *  payload - a string list containing entry & channel
+ * Returns:
+ *  string - the altered entry
+ */
+/datum/tgui_modal/proc/alter_entry(payload)
+	var/entry = payload["entry"]
+	/// No OOC leaks
+	if(payload["channel"] == OOC_CHAN || payload["channel"] == ME_CHAN)
+		return pick(hurt_phrases)
+	/// Sanitizes radio prefixes so users can't game the system (mostly)
```

Comment:
I said this in Discord earlier but I really don't think glorfing should remove radio prefixes, it will just incentivize more ";HELP" and punish people who can't type very fast

Reply:
it is done